getting-started/quickstart-guides/angular.md

```diff
@@ -0,0 +1,834 @@
+* `publicKey` and `authenticationEndpoint` parameters are optional and only needed if you want to use the SDK for client-side file upload. These can be obtained from the [Developer section](https://imagekit.io/dashboard/developer/api-keys) on your ImageKit dashboard.
+
+These details have to be input in the following file.
+
+{% code title="src/app/app.module.ts" %}
+```js
+@NgModule({
+  declarations: [
+    AppComponent
+  ],
+  imports: [
+    BrowserModule,
+    ImagekitioAngularModule.forRoot({
+      publicKey: environment.publicKey,
+      urlEndpoint: environment.urlEndpoint,
+      authenticationEndpoint: environment.authenticationEndpoint
+    })
+  ],
+  providers: [],
+  bootstrap: [AppComponent]
+})
+```
+{% endcode %}
+
+{% hint style="info" %}
+_**Note:**_ _Do not include your _[_API private key_](../../api-reference/api-introduction/api-keys.md#private-key)_ in any client-side code, including this SDK or its initialization. If you pass the `privateKey` parameter while initializing this SDK, it will throw an error._
+{% endhint %}
+
+**ImageKit Components:**
+
+The SDK includes 3 Components and the ability to access the core component:
+
+* [ik-image](https://github.com/imagekit-developer/imagekit-angular#ik-image) for image resizing. This renders a `<img>` tag.
+
+* [ik-video](https://github.com/imagekit-developer/imagekit-angular#ik-video) for video resizing. This renders a `<video>` tag.
+
+* [ik-upload](https://github.com/imagekit-developer/imagekit-angular#ik-upload) for client-side file uploading. This renders a `<input type="file">` tag.
+
+Accessing the underlying [ImageKit javascript SDK](https://github.com/imagekit-developer/imagekit-javascript). See 
+[here](https://github.com/imagekit-developer/imagekit-angular#accessing-imagekit-core-component) for more details.
+
+Note: URL endpoints of each component can be overridden explicitly. [See here for more details](https://github.com/imagekit-developer/imagekit-angular#overriding-urlendpoint)
+
+#### Configure the app for ImageKit:
+
+Let's import the SDK.
+
+{% code title="src/app/app.module.ts" %}
+```js
+import { ImagekitioAngularModule } from 'imagekitio-angular';
+...
+@NgModule({
+  ...
+  imports: [
+    ...
+    ImagekitioAngularModule.forRoot({
+      publicKey: environment.publicKey,
+      urlEndpoint: environment.urlEndpoint,
+      authenticationEndpoint: environment.authenticationEndpoint
+    })
+  ],
+  ...
+})
+...
+```
+{% endcode %}
+
+## **Rendering images**
+
+**Loading image from relative path:**
+
+Let's try using the default image that we have. It should be available at the following URL:
+
+```
+https://ik.imagekit.io/demo/default-image.jpg
+```
+
+Let's fetch and display it! For this, we will use the `ik-image` component.&#x20;
```

Comment:
What is `&#x20;` here? Please remove all such things.

Reply:
Done